lib/awful/layout/suit/spiral.lua

```diff
@@ -11,6 +11,7 @@
 -- Grab environment we need
 local ipairs = ipairs
 local math = math
+local tag = require("awful.tag")
```

Comment:
why?

Reply:
Valid point. Seems I forgot to remove that when cleaning up my changes. My bad.

Reply:
also you need this: https://github.com/awesomeWM/awesome/blob/master/lib/naughty/core.lua#L16